plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -1547,6 +1550,31 @@ private void testReadV2Checkpoint(String resourceName)
         assertQuery("SELECT * FROM " + tableName, "VALUES (1, 2)");
     }
 
+    @Test
+    public void testTypeWideningNotSkippingUnsupportedPartitionColumns()
```

Comment:
Could you add javadoc with link to resource path so that we can confirm the transaction logs quickly? Same for other tests. 

Also, I would move testTypeWideningNotSkippingUnsupportedPartitionColumns and testTypeWideningSkippingUnsupportedColumns under testTypeWidening.